firefox-ios/Client/Frontend/Home/TopSites/DataManagement/UnifiedAds/UnifiedAdsProvider.swift

```diff
@@ -114,7 +117,10 @@ class UnifiedAdsProvider: URLCaching, UnifiedAdsProviderInterface, FeatureFlagga
             guard let self = self else { return }
             switch result {
             case .success(let result):
-                self.cache(response: result.response, for: request, with: result.data)
+                // FXIOS-10798 - URLCache doesn't retrieve from cache if there's an httpBody set on the request
```

Comment:
Can we make a new ticket number for these notes to track setting this code on 🔥  in the new year, or whenever you thought we wouldn't need this code? 😆 (I don't have context on the project so maybe this whole file will get wiped, not sure lol)

Reply:
At some point it is supposed to be wiped entirely, but... we never know lol. So if this feature gets integrated as is in the new homepage code, I'll note this down over there that this is not so great 😅 

The issue I have to track this work is [here](https://mozilla-hub.atlassian.net/browse/FXIOS-10757)